04-materials/02-anatomy-of-extensions.md

```diff
@@ -77,32 +117,373 @@ graph TB
     copier copy --trust https://github.com/jupyterlab/extension-template .
     ```
 
-3. Install it in development mode
+    Be sure to select `frontend-and-server` as the "kind"!
+    Everything else can be left as default.
```

Comment:
GitHub question can be confusing and potentially hard to fix later. Should we instruct them to create a destination GitHub repo as a prereq?

Reply:
Oh, I failed to notice the existing block above
```bash
mkdir my-jupyterlab-extension
cd my-jupyterlab-extension
git init
```
I guess there we can give them gh CLI command

```
gh repo create my-jupyterlab-extension --public --clone
```